openadapt/events.py

```diff
@@ -85,11 +104,23 @@ def get_events(
     return action_events  # , window_events, screenshots
 
 
-def make_parent_event(child, extra=None):
+def make_parent_event(
+    child: models.ActionEvent, extra: dict = None
```

Comment:
Should `extra` be `dict[str, Any]`?

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507